grant-project/zcash-grant-system
# New footer

Closes #285, closes #248.

### What This Does

Updates the footer design and links to go to the right places. Removes about page and makes it redirect instead.

### Steps to Test

* Click all of the links and make sure they go to the right place
* Resize the browser and make sure the footer always looks good
* Go to /about and confirm you're redirected to zfnd.org

## Screenshots

![footer](https://user-images.githubusercontent.com/649992/53833323-3aef2800-3f56-11e9-8c9f-6aa7c014915c.gif)
